src/lib.rs

```diff
@@ -22,6 +22,15 @@
 //!
 //! ## Usage
 //!
+//! This crate is [on crates.io](https://crates.io/crates/mersenne_twister) 
+//! and can be used by adding `mersenne_twister` to the dependencies in your
+//! project's `Cargo.toml`.
+//!
+//! ```
```

Comment:
This block is being interpreted as a Rust doctest, which is making the Travis build fail. Try adding the toml extension to the block:

```
```toml
[dependencies]
...
```

IMO documenting this is fine, but not extremely useful, since this is how most crates are included in a project.

Reply:
Gotcha! Added to make it work for the time of discussion.